content/zh/docs/kitex/Tutorials/advanced-feature/multi_service.md

```diff
@@ -128,18 +141,108 @@ func main() {
 }
 ```
 
-另一种避免 Server 启动错误而不指定回退 Service 的方法是使用`server.WithRefuseTrafficWithoutServiceName`选项。
+另一种避免 Server 启动错误而不指定备用 Service 的方法是使用`server.WithRefuseTrafficWithoutServiceName`选项。
 
-使用此选项，即使您没有为名称冲突的方法指定回退 Service ，启动 Server 时也不会返回错误。
+使用此选项，即使您没有为名称冲突的方法指定备用 Service ，启动 Server 时也不会返回错误。
 
 但在使用此选项时，必须注意以下事项：
 
 当`server.WithRefuseTrafficWithoutServiceName`选项启用时，如果 Server 在以下情况下收到请求，
 则会出现错误消息：“no service name while the server has WithRefuseTrafficWithoutServiceName option enabled”
 
-- 客户端使用较旧的 Kitex 版本(<v0.9.0)，不支持多 Service 功能
-- 请求的传输协议不是 TTHeader ( Kitex pb 的传输协议默认启用 TTHeader )
-- 未设置客户端选项`client.WithMetaHandler(transmeta.ClientTTHeaderHandler())`
+1. 客户端使用较旧的 Kitex 版本(<v0.9.0)，不支持多 Service 功能
+2. 请求的传输协议不是 TTHeader ( Kitex pb 的传输协议默认启用 TTHeader )
+3. 未设置客户端选项`client.WithMetaHandler(transmeta.ClientTTHeaderHandler())`
```

Comment:
@Marina-Sakai `transmeta.ClientTTHeaderHandler` is an object, not a function call.

@Madxf please update each occurrence of `transmeta.ClientTTHeaderHandler()` to `transmeta.ClientTTHeaderHandler` in both CN/EN docs.

Reply:
omg, I'll fix it